A3ServerTool/A3ServerTool.csproj

```diff
@@ -1,33 +1,33 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
-
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0-windows</TargetFramework>
     <UseWPF>true</UseWPF>
     <UseWindowsForms>true</UseWindowsForms>
     <RootNamespace>A3ServerTool</RootNamespace>
     <ApplicationIcon>iconSmall.ico</ApplicationIcon>
-    <Version>1.5.0.2</Version>
+    <Version>1.5.1.0</Version>
   </PropertyGroup>
-
   <ItemGroup>
     <PackageReference Include="MahApps.Metro">
-      <Version>2.3.2</Version>
+      <Version>2.4.9</Version>
     </PackageReference>
     <PackageReference Include="MahApps.Metro.IconPacks.Material">
-      <Version>4.8.0</Version>
+      <Version>4.11.0</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.Xaml.Behaviors.Wpf">
-      <Version>1.1.19</Version>
+      <Version>1.1.39</Version>
     </PackageReference>
     <PackageReference Include="MvvmLightStd10">
       <Version>5.4.1.2</Version>
     </PackageReference>
     <PackageReference Include="Newtonsoft.Json">
       <Version>13.0.1</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers" Version="0.4.336902">
```

Comment:
Could this be deleted as upgrading process already happened in your PR?

Reply:
yep